### PR TITLE
Adding ability to pass exclude and ignore values to action

### PR DESCRIPTION
**Pull request by nchiasson-dgi**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -19,6 +19,13 @@ inputs:
     description: Comma separated list of file extentions to lint.
     required: false
     default: php,module,inc,install,test
+  phpcs-ignore:
+    description: Comma separated list of files to ignore.
+    required: false
+    default: .md
```
Comment (adam-vessey) [inline action.yml:25]: Was spec'd a `*.md` originally... is the `*` relevant?

File: action.yml [hunk 2]
```diff
@@ -34,11 +41,19 @@ runs:
...
         ${{ github.action_path }}/vendor/bin/phpcs -s --standard=Drupal,DrupalPractice
-        --extensions=${{ inputs.extensions }} ${{ inputs.path }} --ignore=*.md
-    - name: Copy-Pasta
+        --extensions=${{ inputs.extensions }} ${{ inputs.path }} --ignore=${{ inputs.phpcs-ignore }}
+    - name: Copy-Pasta - Exclude
+      shell: bash
+      if: ${{ always() && inputs.phpcpd-exclude != '' }}
+      run: >
+        ${{ github.action_path }}/vendor/bin/phpcpd --suffix=${{ inputs.suffix }}
+        --exclude ${{ inputs.phpcpd-exclude }} ${{ inputs.path }}
```
Comment (adam-vessey) [inline action.yml:53]: Generally a good idea to be consistent with how options are specified, like: including or excluding the `=` kind of thing... unless it isn't actually accepted here?

Comment (nchiasson-dgi) [reply action.yml:53]: Couldn't find consistent documentation on using either `exclude` or `exclude=`, after digging into the actual code looks like `exclude=` is used and `=` is used in other options, so including it for consistency.

File: action.yml (hunk 1 above)
Comment (nchiasson-dgi) [reply action.yml:25]: This didn't seem relevant or affect the testing whether it was there or not, but considering it was originally there and worked properly, including it back.

**Review by adam-vessey — APPROVED** [review]

**Pull request merged**